stdlib/string.pyi

```diff
@@ -64,6 +64,15 @@ class Formatter:
     ) -> LiteralString: ...
     @overload
     def vformat(self, format_string: str, args: Sequence[Any], kwargs: Mapping[str, Any]) -> str: ...
+    def _vformat(
+        self,
+        format_string: str,
+        args: Sequence[Any],
+        kwargs: Mapping[str, Any],
+        used_args: Sequence[int | str],
+        recursion_depth: int,
+        auto_arg_index: int = ...,
+    ) -> str | bool: ...
```

Comment:
```suggestion
    ) -> str: ...
```
Don't see anything in the code that returns a bool.

Reply:
Thanks, I misread.